cherryontech/unmatched
# Feature/um 29 nav buttons

## Description

This PR redesigns the navbar to fit  figma design. It also adds the required hover to the buttons. active button issue is now resolved!. note:This branch has been merged to the responsive design branch.


## Acceptance Criteria
Default Button State:

- [x]  When not selected, navigation buttons ('About Us' & 'Resources') should display a solid red background by default.

Selected Button State:

- [x] The button representing the currently active/selected page should be filled with a gradient, as depicted in the provided button image.

Hover State:

- [x] Upon hover, any unselected button should transition to a different shade to indicate interactivity without affecting the selected button's gradient fill.

Persistent Selection Indication:

- [x] Once a user navigates to a page, the corresponding button must retain the gradient fill throughout the session or until a different navigation button is selected.

Transition Animation:

- [x] Implement a smooth transition effect when the button state changes from unselected to selected and vice versa.

## What type of PR is this? (check all applicable)


|     | Type                       |
| --- | -------------------------- |
|     | :sparkles: New feature     |
|  ✓   | 🎨 Style                   |
|     | :hammer: Refactoring       |
|     | 🔥 Performance Improvements|
|     | :bug: Bug fix              |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Mobile & Desktop Screenshots

<img width="1292" alt="Screenshot 2023-11-19 at 9 19 00 AM" src="https://github.com/cherryontech/unmatched/assets/110904882/2bb19a90-68f6-4a00-92fb-3e90efd48e7c">

## What I learned


## Testing steps

Switch to this branch with the following command: git switch feature/UM-29-nav-buttons
Run the following scripts: npm run dev to start Vite's dev build 
View the app in browser (the npm run dev script will return the url you can see the app at)
Hover over the navigation buttons to see hover effect, hover over the 'Learn more' buttons on the cards
Navigate to the bio page of one card. Hover over the "return to unmatched voices" button and the "open resource" button
Note Navbar responsiveness is maintained.
On each bio page, read through the copy, check for typos, makes suggestions, and check the links to make sure they work.


## Added to documentation?

|     | Type                       |
| --- | -------------------------- |
|     | 📜 README.md    |
| ✓  | 🙅 no documentation needed    |



  ![trust me](https://media.giphy.com/media/oC9V7U5mWbUixtyA21/giphy.gif)
